### PR TITLE
2025-overprivileged-nhi

### PR DESCRIPTION
**Pull request by bar-orca**
Formatted and edited version of risk #5

### CODE REVIEW

**Comment by TalAstrix** [comment]
Looks good! Added one small comment, fix and I'll approve
I liked adding links to the data sources and the additional references.

File: 2025/docs/5-overprivileged-nhi.md [hunk 1]
```diff
@@ -1 +1,55 @@
...
+| Exploitability - **Hard**                                                                                                                                                                        | Prevalence - **Widespread** : Detectability - **Hard**                                                                        | Technical **Severe** : Business **Specific** |
+| Successfully exploiting an overprivileged NHI requires the threat agent to first gain access to the environment. Therefore, Overprivileged NHI is dependent on a separate initial access vector. | NHIs are very commonly over-privileged because right-sizing privileges for NHIs is a very difficult and time-consuming task. Detecting Overprivileged NHI is difficult given the challenges in understanding which of the NHI's privileges are actually being used by the underlying application.| Detecting Overprivileged NHI is difficult given the challenges in understanding which of the NHI's privileges are actually being used by the underlying application.|
```
[the remark this answers is not in the document]
Comment (bar-orca) [reply 2025/docs/5-overprivileged-nhi.md]: Good catch! Resolved.

[the remark this answers is not in the document]
Comment (TalAstrix) [reply 2025/docs/5-overprivileged-nhi.md]: Prevalence and Detectability are in the same cell and you need to add another cell for the Technical impact

**Comment by TalAstrix** [comment]
There's the technical impact description still missing

File: 2025/docs/5-overprivileged-nhi.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bar-orca) [reply 2025/docs/5-overprivileged-nhi.md]: Oh I did a mixup of the cells. Hope it's ok now

**Review by TalAstrix — APPROVED** [review]
Looks good!

**Pull request merged**